### DISCUSSION TITLE
Resume Subhranil Nandy

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/resume-pitch#107] [Q&A]
[subsd.pdf](https://github.com/user-attachments/files/17289590/subsd.pdf)
[ATS Version of CV (30).docx](https://github.com/user-attachments/files/17289593/ATS.Version.of.CV.30.docx)

**Comment by Subhranil2004** [comment]
Thanks a lot for the suggestions. 
Kindly have a look at my updated resume and give further suggestions. [Google drive link](https://drive.google.com/file/d/1FrghAuLZyePsC8M88CFMITo1ZsbHEUAO/view?usp=sharing) 

Some points that I need to inform (some based on the video suggestions you gave) :
- Both of my experiences have been voluntary till now. Coz I didn't receive any stipend for my research internship
- My hobbies/interests are mentioned at the bottom part
- I have used the resume template from FlowCV website. I don't know why the ATS version of the doc that you provided above has letters of a single word separated by a space. Coz even if I copy-paste my entire resume, the words don't get separated, at least.

PS: Apologies for the late reply. Have been a bit busy recently.

**Reply by sanjay-kv** [reply]
This might take some time, could you comment it on this coming sat.

**Reply by Subhranil2004** [reply]
@sanjay-kv Could you pl check out my resume once you get time?

**Reply by revanth-bl** [reply]
It looks much cleaner and better structured now. Voluntary experience still counts as good experience, especially for research work, so it’s good that you mentioned it clearly. The FlowCV template looks professional, and the ATS spacing issue is probably just a formatting/PDF conversion thing if the text copies normally Overall, nice improvement

**Comment by PranjalTrivedi** [comment]
@Subhranil2004 try to make your resume simple ATS friendly.

**Reply by Subhranil2004** [reply]
I had updated my resume after that. Kindly have a look.
[Drive link](https://drive.google.com/file/d/1FrghAuLZyePsC8M88CFMITo1ZsbHEUAO/view?usp=sharing)
